contest/views.py

```diff
@@ -156,7 +163,7 @@ def ask(request):
         contest_obj = Contest.objects.get(pk = contest)
     except:
         logger.warning('Clarification: Can not create Clarification!')
```

Comment:
I think we need a log message with more information, such as contest id doesn't exist or something like that.